azure_functions_worker/utils/dependency.py

```diff
@@ -210,17 +210,17 @@ def reload_azure_google_namespace_from_worker_deps(cls):
         """
         # Reload package namespaces for customer's libraries
         packages_to_reload = ['azure', 'google']
+        packages_reloaded = []
         for p in packages_to_reload:
             try:
-                logger.info('Reloading %s module', p)
                 importlib.reload(sys.modules[p])
+                packages_reloaded.append(p)
             except Exception as ex:
                 logger.info('Unable to reload %s: \n%s', p, ex)
-            logger.info('Reloaded %s module', p)
+
+        logger.info(f'Reloaded modules: {",".join(packages_reloaded)}')
```

Comment:
We need the `azure.functions` log as well. Right?

Reply:
Also, rather than f-string, use the %s method as its a little more performant.

Reply:
We don't need the "Reloading" log. If it's successful, The "Reloaded" log will have both the packages else the "Unable to reload" log will be shown,



> Also, rather than f-string, use the %s method as its a little more performant.

Also, f-strings in Python are generally faster than the older string formatting methods such as using the "%". T f-strings are evaluated at runtime and don't require additional parsing steps